src/Storage/Storage/Model/Contract/StorageBlobManagement.cs

```diff
@@ -66,6 +66,10 @@ private CloudBlobClient BlobClient
                 return this.blobClient;
             }
         }
+        public bool IsSasWithOAuthCredential()
```

Comment:
Please add if the Track2oauthtoken is null check in new line #186 in this file , in function `public BlobServiceClient GetBlobServiceClient(BlobClientOptions options = null)`